### DISCUSSION TITLE
Should I update this extension with git pull if I'm on another distro (not pop!_OS)?

### DISCUSSION THREAD

**Discussion by rdong8** [pop-os/shell#1254] [Q&A] [answered]
Or do I need to delete it and reinstall it? And how would I do that, can I just remove the directory that it created?

**Comment by ghost** [comment, marked as answer]
In short you can just `git pull` and `make local-install`.

To answer more thoroughly, if you look at the `local-install:` target of the `Makefile`, it has a prerequisite `install`, and if you look at the [first command](https://github.com/pop-os/shell/blob/afb4f12df1b8ca24dda6aff7ea157bedb8fff208/Makefile#L63) of the `install:` target you'll see how the same code as the `uninstall` target is executed, which effectively first removes the old build and the subsequent commands replace it with the new one.

Hope this helped. ^^
